gnovm/pkg/gnolang/uverse_test.go

```diff
@@ -0,0 +1,165 @@
+package gnolang
+
+import (
+	"testing"
+)
+
+func TestPrintlnWithNilValue(t *testing.T) {
```

Comment:
Table driven tests should be used in situations like this where the input differs but the rest of the test is the same.  Here is an example of how to write them: https://dave.cheney.net/2019/05/07/prefer-table-driven-tests